docs/src/pages/styles/advanced/advanced.md

```diff
@@ -553,6 +553,32 @@ This vulnerability would allow the attacker to execute anything. However, with a
 
 You can read more about CSP on the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
 
+### Does Material UI support CSP?
+
+Most Material UI components support a strict CSP. Specifically, this means you will not need to use `'unsafe-inline'`. However, some components still use inline styles. If you use these components you must set `'unsafe-inline'` in the `style-src` directive.
```

Comment:
> Most

I don't think this is true. The list you provided only looks at components who use `styles` directly. It doesn't include components that import these components e.g. `Button` imports `ButtonBase`.